temporal-sdk/src/test/java/io/temporal/workflow/DescribeTest.java

```diff
@@ -154,9 +151,6 @@ public void testSuccessfulActivity() throws InterruptedException {
             // Heads up! We're asserting that heartbeat time == started time, which should be true
             // before the heartbeat
             .setLastHeartbeatTime(actual.getLastStartedTime())
-            // this ends up being a dummy value, but if it weren't, we still wouldn't expect to know
-            // it.
-            .setLastWorkerIdentity(actual.getLastWorkerIdentity())
```

Comment:
Not supposed to have identity here, because the task hasn't been picked up by the worker yet.